src/scripts/validation.js

```diff
@@ -52,6 +52,11 @@ const toggleButtonState = (inputList, buttonElement, config) => {
   }
 };
 
+export const disableSubmitButton = (buttonElement, config) => {
```

Comment:
Please, use `disableSubmitButton` inside function `toggleButtonState` not to duplicate the code